package.json

```diff
@@ -35,15 +35,16 @@
     "remove-leading-slash": "^1.0.1",
     "remove-trailing-slash": "^0.1.0",
     "search-a-tags": "^1.0.1",
-    "search-img-tags": "^1.0.0"
+    "search-img-tags": "^1.0.0",
+    "yarn": "^1.19.1"
```

Comment:
can you remove this yarn.
we don't need it

Reply:
Removed them. Please review it now